src/app/dim-ui/ItemCategoryIcon.tsx

```diff
@@ -17,19 +16,11 @@ export function ArmorSlotIcon({ item, className }: { item: DimItem; className?:
   );
 }
 
-// don't invert these. they're perfect as-is.
-const imagesToLeaveAlone = [kineticSlot];
-
 export function WeaponSlotIcon({ item, className }: { item: DimItem; className?: string }) {
   const slotIcon = getWeaponSlotSvgIcon(item)!;
   return (
     <PressTip minimal elementType="span" tooltip={item.bucket.name}>
-      <img
-        src={slotIcon}
-        className={clsx(styles.itemCategoryIcon, className, {
-          [styles.dontInvert]: imagesToLeaveAlone.includes(slotIcon),
-        })}
-      />
+      <img src={slotIcon} className={clsx(styles.itemCategoryIcon, styles.dontInvert, className)} />
```

Comment:
If we're always applying `dontInvert` can we just roll them into `itemCategoryIcon`?

Reply:
`itemCategoryIcon` is also used for armor & weapon type icons, which **are** inverted, so we can't merge them in this case.